models/iaf_psc_exp.h

```diff
@@ -85,11 +84,25 @@ optimization levels. A future version of iaf_psc_exp will probably
 address the problem of efficient usage of appropriate vector and
 matrix objects.
 
-Parameters:
+If tau_m is very close to tau_syn_ex or tau_syn_in, the model
+will numerically behave as if tau_m is equal to tau_syn_ex or
+tau_syn_in, respectively, to avoid numerical instabilities.
+For details, please see IAF_neurons_singularity.ipynb in the
+NEST source code (docs/model_details).
+
+iaf_psc_exp can handle current input in two ways: Current input
+through receptor_type 0 are handled as stepwise constant current
+input as in other iaf models, i.e., this current directly enters
+the membrane potential equation. Current input through
+receptor_type 1, in contrast, is filtered through an exponential
+kernel with the time constant of the excitatory synapse,
+tau_syn_ex. For an example application, see [6].
```

Comment:
You are pointing here to reference [6] (without trailing `_`), but the reference list has only four entries. Maybe some old version of the model reveals what [6] pointed to once upon a time ;).

Reply:
I checked the file's history, and it seems to have been pointing to ``[4]`` in the past: https://github.com/nest/nest-simulator/blob/853911abebdd3d893099f719c4db1d35d0353c96/models/iaf_psc_exp.h